src/cupbearer/data/pytorch.py

```diff
@@ -5,31 +5,60 @@
 from cupbearer.utils.utils import get_object, mutable_field
 
 from . import DatasetConfig
-from ._shared import ToNumpy, Transform
+from ._shared import Resize, ToNumpy, Transform
 
 
 @dataclass(kw_only=True)
 class PytorchConfig(DatasetConfig):
     name: str
+    num_classes: int
```

Comment:
Shouldn't this be in `DatasetConfig` instead? In practice, the `train_classifier` script will probably only be called with `PytorchConfig` but unless we actually change the type annotations/CLI options to reflect that, I think we need `num_classes` everywhere.